lib/common/benchmarkModelGenerator.js

```diff
@@ -0,0 +1,408 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const { Blob } = require('buffer');
+const {
+    uniqueNamesGenerator, adjectives, colors, animals
+} = require('unique-names-generator');
+
+const DEFAULT_CONCERTO_METAMODEL_VERSION = '1.0.0';
+const DEFAULT_MODEL_NAMESPACE = 'big.benchmark.model';
```

Comment:
Nit, I think that this code is much more general-purpose than for generating big models, or models for benchmarking. Can you soften this a little?